utils/constants.ts

```diff
@@ -26,5 +26,10 @@ export const TOKEN_REGISTRY_CONTRACT_CACHE_KEY = "TokenRegistry";
 export const TOKEN_TRANSFER_PROXY_CONTRACT_CACHE_KEY = "TokenTransferProxy";
 export const TERMS_CONTRACT_REGISTRY_CONTRACT_CACHE_KEY = "TermsContractRegistry";
 export const COLLATERALIZED_SIMPLE_INTEREST_TERMS_CONTRACT_CACHE_KEY =
-    "CollateralizedSimpleInterestTermsContract";
+  "CollateralizedSimpleInterestTermsContract";
 export const COLLATERALIZER_CONTRACT_CACHE_KEY = "CollateralizerContract";
+
+export const TERMS_CONTRACT_TYPES = {
+  COLLATERALIZED_SIMPLE_INTEREST_LOAN: "CollateralizedSimpleInterestLoan",
+  SIMPLE_INTEREST_LOAN: "SimpleInterestLoan",
+};
```

Comment:
Note that we use 4-space tabs, not 2-space tabs -- I'm surprised Hound isn't picking this up.

Reply:
Done.